helm/adelphi/config/grafana/nb4-new-dashboard.json

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
This is the default dashboard that comes with NB. We'll soon replace it with something that makes more sense for Adelphi, like some kind of side-by-side comparison with sensitive performance metrics. New ticket coming up for this.

Reply:
https://github.com/datastax/adelphi/issues/91